airflow/jobs/scheduler_job.py

```diff
@@ -982,10 +982,13 @@ def _find_executable_task_instances(self, simple_dag_bag, states, session=None):
 
             Stats.gauge('pool.starving_tasks.{pool_name}'.format(pool_name=pool_name),
                         num_starving_tasks)
+            slots_stats = pools[pool_name].slots_stats()
             Stats.gauge('pool.open_slots.{pool_name}'.format(pool_name=pool_name),
-                        pools[pool_name].open_slots())
+                        slots_stats["open"])
```

Comment:
```suggestion
                        open_slots)
```

Possibly. Given this local variable is what the scheduler thinks the pool open slots are, it would make sense to report the metric on that, not based on the DB value.

Reply:
Yes, I think we haven't yet updated the DB with the new pool usages yet, so the DB value won't be correct. I think. Are you able to confirm or deny this?

Reply:
yes, db hasn't been updated in DB at this point yet. i will reply to this in my follow up comment below as well.